usaspending_api/etl/award_helpers.py

```diff
@@ -119,3 +119,35 @@ def update_contract_awards(award_tuple=None):
         rows = cursor.rowcount
 
     return rows
+
+
+def update_award_subawards(award_tuple=None):
+    """
+    Updates awards' subaward counts and totals
+    """
+    # Sum and count subaward_amounts
+    sql_sub_totals = (
+        'subaward_totals AS ('
+        'SELECT award_id, SUM(amount) AS total_subaward_amount, COUNT(*) AS subaward_count '
+        'FROM awards_subaward ')
+    if award_tuple:
+        sql_sub_totals += 'WHERE award_id IN %s '
```

Comment:
Not for these commands, no. I was just using the code we currently have from prior helpers. We should look to refactor/tidy them up as time permits because I agree it is a bit fast and loose.